assets/chat/js/regex.js

```diff
@@ -1,3 +1,21 @@
+const regexslashcmd = /^\/([a-z0-9]+)[\s]?/i;
+const regextime = /(\d+(?:\.\d*)?)([a-z]+)?/gi;
+const nickmessageregex = /(?=@?)(\w{3,20})/g;
+const nickregex = /^[a-zA-Z0-9_]{3,20}$/;
+const nsfwnsflregex = /\b(?:NSFL|NSFW)\b/i;
```

Comment:
This regex and its related logic seems pointless because we already check for `nsfw` and `nsfl` separately in `ignored()`. I think its safe to remove.